ldk/javascript/src/types/oliveHelps/index.d.ts

```diff
@@ -423,6 +424,11 @@ declare namespace OliveHelps {
     tooltip?: string;
   };
 
+  type SectionTitle = Component<'sectionTitle'> & {
+    body: string;
+    textalign?: TextAlign;
+  };
```

Comment:
This might just be me still learning - but in your bitbucket PR to sidekick, it looks like SectionTitle only has options for the body attribute exposed?

Reply:
Resolved